RevenueCatUI/Templates/Components/TemplateComponentsView.swift

```diff
@@ -4,18 +4,32 @@
 //
 //  Created by Josh Holtz on 6/11/24.
 //
-// swiftlint:disable missing_docs todo
+// swiftlint:disable missing_docs
 
 import RevenueCat
 import SwiftUI
 
 #if PAYWALL_COMPONENTS
+
+class PaywallState: ObservableObject {
+
+    @Published var selectedPackage: Package?
+
+    func select(package: Package) {
+        self.selectedPackage = package
+    }
+
+}
+
 @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
 public struct TemplateComponentsView: View {
```

Comment:
Noticed this was public, since it's behind the `PAYWALL_COMPONENTS` flag, maybe we can also make it internal now?